src/20_Components/amp-analytics.html

```diff
@@ -138,20 +138,20 @@
           </amp-img>
       </div>
       <!--
-    The `elementview` event is issued when the image becomes visible -->
+    The `elementShown` event is issued when the image becomes visible -->
   <amp-analytics>
       <script type="application/json">
       {
         "requests": {
-          "elementview": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user=[[.]]&account=ampbyexample&event=${eventId}"
+          "elementShown": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user=[[.]]&account=ampbyexample&event=${eventId}"
         },
         "triggers": {
           "defaultPageview": {
             "on": "visible",
-            "request": "elementview",
+            "request": "elementShown",
             "selector": "#an-image",
             "vars": {
-              "eventId": "element-ui"
+              "eventId": "imageview"
```

Comment:
This `imageview` is very generic which makes it hard to associate the different events. If you look at the other samples, I'm using concrete events (e.g. `catview`). Maybe: `imageShownAfterButtonClick`.